templates/shortcodes/references.md

```diff
@@ -5,44 +5,44 @@
 [contributing]: https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md
 [email]: mailto:support@organicmaps.app
 [exodus]: https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/
-[trackercontrol]: https://ios.trackercontrol.org/analysis/app.organicmaps
 [facebook]: https://facebook.com/OrganicMaps
 [fdroid]: https://f-droid.org/{{ lang | truncate(length=2, end="") }}/packages/app.organicmaps/ "{{ trans(key='install-fdroid', lang=lang) }}"
 [firebase]: https://appdistribution.firebase.dev/i/2f0fee463107b137
 [flatpak]: https://flathub.org/apps/details/app.organicmaps.desktop
 [FOSStodon]: https://fosstodon.org/@organicmaps "FOSStodon"
 [github]: https://github.com/organicmaps/organicmaps
-[googleplay]: https://play.google.com/store/apps/details?id=app.organicmaps&hl={{ lang }} "{{ trans(key='install-googleplay', lang=lang) }}"
 [github issues]: https://github.com/organicmaps/organicmaps/issues "GitHub Issues"
+[googleplay]: https://play.google.com/store/apps/details?id=app.organicmaps&hl={{ lang }} "{{ trans(key='install-googleplay', lang=lang) }}"
 [googleplay_review]: market://details?id=app.organicmaps
 [gpx_issue]: https://github.com/organicmaps/organicmaps/issues/624
 [ideas]: https://github.com/organicmaps/organicmaps/discussions/categories/ideas
 [instagram]: https://instagram.com/organicmaps.app/
 [issues]: https://github.com/organicmaps/organicmaps/issues
-[LinkedIn]:  https://www.linkedin.com/company/organic-maps/
+[LinkedIn]: https://www.linkedin.com/company/organic-maps/
 [mapswithme]: https://en.wikipedia.org/wiki/Maps.me "Maps.Me"
 [mastodon]: https://mastodon.social/@organicmaps "Mastodon"
 [matrix]: https://matrix.to/#/#organicmaps:matrix.org "Organic Maps on the Matrix"
 [matrix_ru]: https://matrix.to/#/#organicmapsru:matrix.org "Organic Maps в Matrix"
-[mythic_beasts]: https://www.mythic-beasts.com/
 [mythic_beasts_donation]: https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/
+[mythic_beasts]: https://www.mythic-beasts.com/
 [openstreetmap]: https://www.openstreetmap.org/about "OpenStreetMap"
 [public_transport_validator]: https://cdn.organicmaps.app/subway/
 [reddit]: https://www.reddit.com/r/organicmaps/
 [repology]: https://repology.org/project/organicmaps/versions
-[stripe]: https://donate.organicmaps.app/ "Donate"
 [stripe_chf]: https://donate.organicmaps.app/?currency=chf "Donate in CHF"
 [stripe_eur]: https://donate.organicmaps.app/?currency=eur "Donate in EUR"
 [stripe_gbp]: https://donate.organicmaps.app/?currency=gbp "Donate in GBP"
+[stripe]: https://donate.organicmaps.app/ "Donate"
```

Comment:
@biodranik your collation sorting - but strange that [stripe] is in the middle

Reply:
That looks like a bug in their algo/implementation.

Reply:
For consistency, a shorter string should be at the top, not at the bottom (see other examples in this file).